### PR TITLE
DEVDOCS-2084: Dev Center | Theme Docs | Translation Keys

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2084](https://jira.bigcommerce.com/browse/DEVDOCS-2084)

## What changed?
* applied sentence case
* applied code font
* updated links
* updated code example
* changed table formatting from html to markdown

### CODE REVIEW

File: docs/stencil-docs/localization/translation-keys.md [hunk 1]
```diff
@@ -258,7 +237,7 @@ lineNumbers: true
 
 ```
 
-In the excerpts below, you can see that the default theme’s <span class="fp">/lang/en.json</span> file includes matches, and value definitions, for each of the translation keys in the Handlebars helpers above.
+In the example below, the default theme’s `/lang/en.json` file includes matches and value definitions for each of the translation keys.
```
Comment (aglensmith) [inline docs/stencil-docs/localization/translation-keys.md:240]: Replace `/lang/en.json`  with `lang/en.json`. 
```suggestion
In the example below, the default theme’s `lang/en.json` file includes matches and value definitions for each of the translation keys.

```

File: docs/stencil-docs/localization/translation-keys.md [hunk 2]
```diff
@@ -275,47 +254,47 @@ lineNumbers: true
...
+    },
+}
 ```
 
-Translation files for other languages would use the same format to define these keys’ values in their respective languages.
+Translation files for other languages would use a similar format to define key-value pairs in their respective languages.
 
 ## Resources
 
 ### Additional Resources
```
Comment (aglensmith) [inline docs/stencil-docs/localization/translation-keys.md:296]:
```suggestion
### Additional resources

```

File: docs/stencil-docs/localization/translation-keys.md [hunk 3]
```diff
@@ -3,101 +3,77 @@
 <div class="otp" id="no-index">
 
 ### On This Page
```
Comment (aglensmith) [inline docs/stencil-docs/localization/translation-keys.md:5]:
```suggestion
### On this page

```

File: docs/stencil-docs/localization/translation-keys.md [hunk 4]
```diff
@@ -2,102 +2,79 @@
...
+- [Invoking a translation key](#invoking-a-translation-key)
+- [Localization example](#localization-example)
 - [Resources](#resources)
 
 </div> 
 
 Translation keys exist in JSON files and are invoked based on the user's browser language. With a Stencil theme, you can define multiple translations for each theme based on a predefined schema.
 
-BigCommerce does not provide translations for the theme’s content, however, you have the option of localizing your themes for desired target languages, based on the provided schema. Theme developers are not required to localize or translate a theme in order to make it work.
+BigCommerce does not provide translations for the theme’s content; however, you have the option of localizing your themes for desired target languages based on the provided schema. Theme developers are not required to localize or translate a theme in order to make it work.
```
Comment (uwikaiddi) [inline docs/stencil-docs/localization/translation-keys.md:19]:
```suggestion
BigCommerce does not provide translations for the theme’s content. However, you have the option of localizing your themes for desired target languages based on the provided schema. Theme developers are not required to localize or translate a theme in order to make it work.

```

File: docs/stencil-docs/localization/translation-keys.md [hunk 5]
```diff
@@ -2,102 +2,79 @@
...
-## Localization File Structure
-
-The <span class="fp">.json</span> translation files have a straightforward structure of key-value pairs. Here is an example, from Cornerstone's <span class="fn">en.json</span> file:
+## Required subdirectory
+
+Within each Stencil theme, a top level `/lang/` subdirectory is reserved for localization and internationalization. For a theme to function properly, both the `/lang/` subdirectory and the `/lang/en.json` file, which contains the English-language defaults, must be present.
+
+You can localize a theme by providing other appropriate JSON translation files in the `/lang` subdirectory. In these files, you would define key-value pairs corresponding to the text blocks in your theme's templates. 
+
+One JSON file is required for each language that you want to support. (These can include non-U.S. versions of English, each with their own spellings.)
```
Comment (uwikaiddi) [inline docs/stencil-docs/localization/translation-keys.md:48]:
```suggestion
One JSON file is required for each language that you want to support. These can include non-U.S. versions of English, each with their own spellings.

```

File: docs/stencil-docs/localization/translation-keys.md [hunk 6]
```diff
@@ -2,102 +2,79 @@
...
+For more examples of frequently-used codes, see [ISO 639-2 Codes for the Representation of Names of Languages](http://www.loc.gov/standards/iso639-2/php/code_list.php).
+
+## Localization file structure
+
+The JSON translation files are structured as key-value pairs. Here is an example from Cornerstone’s `en.json` file: 
 
 `"welcome_back": "Welcome back, {name}"`
 
-Above, `"welcome_back"` is an arbitrary key name for a welcome message. The string `"Welcome back, {name}"` is its assigned value for English-language stores.
+* `"welcome_back"` - is an arbitrary key name for a welcome message.
```
Comment (uwikaiddi) [inline docs/stencil-docs/localization/translation-keys.md:72]:
```suggestion
* `"welcome_back"` - An arbitrary key name for a welcome message. 

```

File: docs/stencil-docs/localization/translation-keys.md [hunk 7]
```diff
@@ -2,102 +2,79 @@
...
+
+## Localization file structure
+
+The JSON translation files are structured as key-value pairs. Here is an example from Cornerstone’s `en.json` file: 
 
 `"welcome_back": "Welcome back, {name}"`
 
-Above, `"welcome_back"` is an arbitrary key name for a welcome message. The string `"Welcome back, {name}"` is its assigned value for English-language stores.
+* `"welcome_back"` - is an arbitrary key name for a welcome message. 
+* `"Welcome back, {name}"` - the value assigned for English-language stores.
```
Comment (uwikaiddi) [inline docs/stencil-docs/localization/translation-keys.md:73]:
```suggestion
* `"Welcome back, {name}"` - The value assigned for English-language stores.

```

**Pull request merged**